armeria-backend/zio/src/main/scala/sttp/client3/armeria/zio/zio.scala

```diff
@@ -0,0 +1,66 @@
+package sttp.client3.armeria
```

Comment:
we'd also need the same jvm/zio module, for the `HttpClientZioBackend`

Reply:
Done.